### ISSUE TITLE
Is iOS's three finger zoom defined as "assistive technology" on native mobile apps?

### ISSUE THREAD

**Issue by kylenowcrylater** [w3c/wcag#4717]
Hello! I am struggling to understand the definition of "assistive technology" on native mobile. 

Comments by @maryjom in w3c/wcag2ict#4 suggest the native magnifier on iOS is now a valid way to pass 1.4.4 for mobile apps:

"features including software provided by the platform that provide a means of enlarging the text 200% (zoom or otherwise) without the loss of content or functionality, meet the intent of this success criteria."

The native three finger zoom in iOS is a software feature provided by the platform (iOS) that enlarges text to 200%. It's mainstream, doesn't require third party software to be installed, etc. I've also read through the W3C meeting minutes and can confirm text is not pixelated when using the feature.

If nothing changes from WCAG, then why say anything at all? I can't wrap my head around any other interpretation of this text. 

If three finger zoom is still assistive technology because it's designed for users with impaired vision, why is iOS' font resizing (notably the "additional _accessibility_ sizes" required to hit 200%) not assistive technology? That clearly falls beyond mainstream usage into specialized targeted functionality that meets the definition of AT.

In my personal opinion these definitions fall apart under scrutiny. 

**Why it matters**

There are various edge cases on mobile where it is impossible to enlarge text to 200% without failing other SC or breaking the layout. 

In mobile apps we don't have the luxury of pinch to zoom that mobile web has, which multiple W3C contributors have mentioned is viable (@patrickhlauke is a great example).

Sometimes we can fall back on Apple's Large Content Viewer or similar, but in a sufficiently complex and large mobile app there will be edge cases where even _that_ is not viable. 

Anyways thank you in advance for clarifying. I have done a deep dive into lots of WC3/WCAG material and I am still really struggling with this definition.

**Comment by maryjom** [comment]
This issue was discussed in the 30 October meeting and we decided not to make any changes to the WCAG2ICT document for this issue. WCAG2ICT cannot make normative changes to the definition of assistive technology or to the SC language to assist in clarifying this issue. The issue also exists with WCAG, where we believe the intent was that assistive technology was 3rd-party AT that would need to be installed and any features built into the ICT that provide similar functionality to 3rd party AT would not be considered AT.

We think this issue should be moved to the WCAG 2.2 issue backlog for them to address it.

**Comment by mraccess77** [comment]
I consider three finger zoom to be an assistive technology because it is magnifying at a different layer than the application - it's very different than the large text setting which would actually enlarge the text within the app. Instead it's at the platform level enlarging the screen.

I consider the larger text settings to NOT be assistive technology because they are available in display to all users and they can be actioned directly by the app and change the size of the text in the app if coded to support them and not on a separate layer outside the application.

**Comment by kylenowcrylater** [comment]
Yet three finger zoom is also available to all users. Both zoom and dynamic text are features provided by the platform. The only real meaningful difference IMO is font resizing can require some action from the app developer, three finger zoom they will always get for free. But that’s also the case with pinch to zoom which contributors are on record saying normatively passes the SC

**Comment by patrickhlauke** [comment]
just a reminder/note that, at least for now in WCAG 2.x, native apps are out of scope. and yes, while three-finger zoom is available to all users, they need to explicitly turn it on in their mobile OS' accessibility settings, compared to pinch zooming which - if we're talking web content, which is what WCAG 2.x is about - is just available by default.

**Comment by GreggVan** [comment]
yes  three finger zoom is an accessibility feature with an inaccessible activation method (if other methods for activating were not provided(

**Comment by GreggVan** [comment]
NOTE:   Three finger zoom would not meet the "single pointer"  nor the "keyboard only" provisions -- but as long as there are OTHER ways to activate the zoom feature including   keyboard only approach and preferably also a single pointer method.   I think that both exist for iOS.    

Can anyone comment on Android?

**Comment by patrickhlauke** [comment]
the single pointer/keyboard aspects relate to the user agent in this case (so, a UAAG concern, not a WCAG one), we're not talking about author-provided methods. so think they're orthogonal to the whole discussion here

**Comment by kylenowcrylater** [comment]
Three finger zoom has to be enabled at the OS level by the user, but that’s also the case for increased font size right which passes the SC.

The case for three finger zoom failing a different gesture-based SC does make sense for sure. I think there’s still an issue with the definition of assistive technology overall that will continue to be an issue though. 

Totally hear you that WCAG is web. What seems to be happening though is the other working groups keep deferring back to WCAG to make decisions. 

If that’s true, perhaps those groups need some autonomy to reinterpret or extend what counts as a normative pass/fail of the SC, or the web working group needs to be willing to work on tasks that aren’t strictly web.

Until then I’m curious how this SC will pan out legally. Seems like a company could make a pretty good argument that they can’t be fined under the EAA for not supporting text resizing.

**Comment by mraccess77** [comment]
When WCAG was created magnification existed in platforms like Windows. It was more limited them to a lens - but the group still chose to indicate that assistive technology couldn't be relied upon to meet the criterion.  It was never accepted that Windows magnifier could meet the requirements - so why now are we saying that Windows or MacOS would suddenly now be used to meet the requirements? 

Platform magnification is not within the browser or page - it's external to the browser and thus it's own user agent - meaning it's an assistive technology.  Assistive technology is a broad term and isn't limited to software from 3rd parties only.

The large text settings are different because it's the application or website that is responding to those settings and not those settings overriding the website.

**Comment by kylenowcrylater** [comment]
Thanks for the explanation, that user agent distinction makes more sense. I suppose we could treat the Large Content Viewer similarly where the application is utilizing it, nothing is being overridden. 

As for why this has come up, I couldn’t tell you. I’m responding to the content that the WCAG2ICT had written and trying to understand what they meant
